src/features/categories/categoriesSlice.js

```diff
@@ -0,0 +1,20 @@
+import { createSlice } from '@reduxjs/toolkit';
```

Comment:
- [ ] According to project requirements, the structure of the files should follow the `ducks pattern`, please kindly move categoriesSlice.js to `/src/redux/categories/categoriesSlice.js`